src/scrape_up/github/users.py

```diff
@@ -662,7 +663,26 @@ def get_open_issues(self):
             return None
 
         return open_issues
+    
+    def get_years_active(self): 
+        """
+        Class - `Users`\n
+        Example -\n
+        ```python
+        user = github.User(username="nikhil25803")
+        years_active = user.get_years_active()
+        ```
+        """
+        page = self.__scrape_page()
+        try:
+            ul_element = page.find("ul", class_="filter-list small")
+            li_element = ul_element.find_all("li")
 
+            years_active = len(li_element)
+
+            return years_active 
+        except:
+            return None
```

Comment:
This change is also added in a sperate PR I guess, I will check that one and update there as well.